<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <title>Soel Micheletti</title>
  <meta name="description" content="Soel Micheletti's Portfolio.">

  <!-- Fonts and Icons -->
  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons" />
  <link rel="shortcut icon" href="/assets/img/favicon.png">
  
  <!-- CSS Files -->
  <link rel="stylesheet" href="/assets/css/all.min.css">
  <link rel="stylesheet" href="/assets/css/academicons.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/projects/">

  <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"/>
  
</head>
<body>
  <!-- Header -->
  <nav id="navbar" class="navbar fixed-top navbar-expand-md grey lighten-5 z-depth-1 navbar-light">
    <div class="container-fluid p-0">
      
        <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Soel</span> Micheletti</a>
      
      <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <li class="nav-item ">
            <a class="nav-link" href="/">
              About
              
            </a>
          </li>


            
              <li class="nav-item">
                  <a class="nav-link" href="/projects/">
                    Projects
                  </a>
              </li>
            
              <li class="nav-item navbar-active font-weight-bold">
                <a class="nav-link" href="/publications/">
                  Publications
                  <span class="sr-only">(current)</span>
                </a>
              </li>
            
              <li class="nav-item ">
                  <a class="nav-link" href="/teaching/">
                    Teaching
                  </a>
              </li>
            
          
            
               <li class="nav-item ">
                  <a class="nav-link" href="/assets/pdf/CV_long.pdf">
                    CV
                  </a>
               </li> 
          
        </ul>
      </div>
    </div>
  </nav>

  <!-- Scrolling Progress Bar -->
  <progress id="progress" value="0">
    <div class="progress-container">
      <span class="progress-bar"></span>
    </div>
  </progress>

  <!-- Content -->
  <div class="content">
    
    <h1>Publications</h1>
  
    <div class="row m-0 p-0">
      <div class="col-sm-11 p-0">

        <!--COBRA-->
        <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
          <div class="col-sm-1 p-0 abbr">
            <a class="badge font-weight-bold danger-color-dark align-middle" style="width: 53px;" href="https://www.biorxiv.org/content/10.1101/2023.12.28.573533v1.abstract" target="_blank">
            BioRxiv
            </a>
          </div>
          
          <div class="col-sm-11 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">
            <div id="cobra_short" class="col p-0">
              <h5 class="title mb-0">Higher-order correction of persistent batch effects in
                correlation networks [UNDER REVIEW]
              </h5>
              <div class="author">
                <em> 
                    <b>Soel Micheletti</b>;
                    Daniel Schlauch;
                    John Quackenbush;
                    Marouen Ben Guebila;
                </em>  
              </div>
  
              <div>
                <p class="periodical font-italic">
                  BioRxiv      
                </p>
              </div>
      
              <div class="col p-0">
                <a class="badge grey waves-effect font-weight-light mr-1" data-toggle="collapse" href="#cobra" role="button" aria-expanded="false" aria-controls="tec2023clim2pm-abstract">Abstract</a>
                <a class="badge grey waves-effect font-weight-light mr-1" href="https://www.biorxiv.org/content/10.1101/2023.12.28.573533v1.abstract" target="_blank">Paper</a>
                <a class="badge grey waves-effect font-weight-light mr-1" href="https://github.com/QuackenbushLab/cobra-experiments" target="_blank">Code</a>
                <a class="badge grey waves-effect font-weight-light mr-1" href="https://netzoo.github.io" target="_blank">Website</a>
              </div>
      
              <div class="col mt-2 p-0">
                <div id="cobra" class="collapse">
                  <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
                    Systems biology methods often rely on correlations in gene expression profiles to infer co-expression networks, commonly used as input for gene regulatory network inference or to identify functional modules of co-expressed or co-regulated genes. While systematic biases, including batch effects, are known to induce spurious associations and confound differential gene expression analyses (DE), the impact of batch effects on gene co-expression has not been fully explored. Methods have been developed to adjust expression values, ensuring conditional independence of mean and variance from batch or other covariates for each gene. These adjustments have been shown to improve the fidelity of DE analysis. However, these methods do not address the potential for spurious differential co-expression (DC) between groups. Consequently, uncorrected, artifactual DC can skew the correlation structure, leading network inference methods that use gene co-expression to identify false, nonbiological associations, even when the input data is corrected using standard batch correction.
                    In this work, we demonstrate the persistence of confounders in covariance after standard batch correction using synthetic and real-world gene expression data examples. Subsequently, we introduce Co-expression Batch Reduction Adjustment (COBRA), a method for computing a batch-corrected gene co-expression matrix based on estimating a conditional covariance matrix. COBRA estimates a reduced set of parameters expressing the co-expression matrix as a function of the sample covariates, allowing control for continuous and categorical covariates. COBRA is computationally efficient, leveraging the inherently modular structure of genomic data to estimate accurate gene regulatory associations and facilitate functional analysis for high-dimensional genomic data.
                  </div>
                </div>
              </div>
              
            </div>
          </div>
        </div></li></ol>

        <!--GIRAFFE POSTER-->
        <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
          <div class="col-sm-1 p-0 abbr">
            <a class="badge font-weight-bold danger-color-dark align-middle" style="width: 60px;" href="https://recomb.org/recomb2024/" target="_blank">
            RECOMB
            </a>
          </div>
          
          <div class="col-sm-11 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">
            <div id="giraffe" class="col p-0">
              <h5 class="title mb-0">Gene-level Inference of Regulatory effects As Factorizations of
                Functions of Expressions (GIRAFFE)
              </h5>
              <div class="author">
                <em> 
                    <b>Soel Micheletti</b>;
                    Alexander Marx;
                    Julia Vogt;
                    John Quackenbush;
                    Jonas Fischer;
                    Panagiotis Mandros;
                </em>  
              </div>
  
              <div>
                <p class="periodical font-italic">
                  Research in Computational Molecular Biology        
                </p>
              </div>
      
              <div class="col p-0">
                <a class="badge grey waves-effect font-weight-light mr-1" data-toggle="collapse" href="#giraffe_poster" role="button" aria-expanded="false" aria-controls="tec2023clim2pm-abstract">Abstract</a>
                <a class="badge grey waves-effect font-weight-light mr-1" href="/assets/pdf/GIRAFFE_poster.pdf" target="_blank">Poster</a>
                <a class="badge grey waves-effect font-weight-light mr-1" href="https://github.com/soelmicheletti/giraffe" target="_blank">Code</a>
              </div>
      
              <div class="col mt-2 p-0">
                <div id="giraffe_poster" class="collapse">
                  <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
                    Accurately estimating gene regulatory mechanisms is crucial to inform our understanding of important cellular processes, yielding new insights into disease progression and intervention design. Exiting methods suffer of one or multiple issues when it comes to interpretability, scalability to the human genome, and flexibility to incorporate covariates of interest such as clinical information about the samples. For instance, state-of-the-art algorithms either do not distinguish between enhancing and inhibitory regulation, or do not scale beyond a few hundreds genes. Moreover, not all existing methods are interpretable. They may incorporate complex non-linear relationships to optimize predictions, yet these are incompatible with human reasoning - a critical component for ensuring the safety, ethics, and accountability of models supporting oncology decisions. 
                    For effective gene regulatory network inference, we propose GIRAFFE, a scalable matrix factorization-based algorithm to jointly infer regulatory effects and transcription factor activities from gene expression data. GIRAFFE integrates prior knowledge about regulation to guide the optimization, yielding an interpretable linear model. Moreover, it can be customized to the requirements of the downstream application by adjusting for variables of interest, such as confounders, or by adding sparsity constraints. We demonstrate the effectiveness of this approach with extensive experiments on synthetic, as well as real world data. Our algorithm outperforms state-of-the-art gene regulatory network inference methods in predicting interactions between transcription factors and target genes. In contrast to existing work, it is able to distinguish between activating and inhibitory effects, yielding plausible results in downstream applications including gene set enrichment analysis. 
                  </div>
                </div>
              </div>
              
            </div>
          </div>
        </div></li></ol>

        <!--COBRA POSTER-->
        <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
          <div class="col-sm-1 p-0 abbr">
            <a class="badge font-weight-bold danger-color-dark align-middle" style="width: 60px;" href="https://recomb.org/recomb2024/" target="_blank">
            RECOMB
            </a>
          </div>
          
          <div class="col-sm-11 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">
            <div id="cobra_short" class="col p-0">
              <h5 class="title mb-0">Higher-order correction of persistent batch effects in
                correlation networks (poster)
              </h5>
              <div class="author">
                <em> 
                    <b>Soel Micheletti</b>;
                    Daniel Schlauch;
                    John Quackenbush;
                    Marouen Ben Guebila;
                </em>  
              </div>
  
              <div>
                <p class="periodical font-italic">
                  Research in Computational Molecular Biology        
                </p>
              </div>
      
              <div class="col p-0">
                <a class="badge grey waves-effect font-weight-light mr-1" data-toggle="collapse" href="#cobra_poster" role="button" aria-expanded="false" aria-controls="tec2023clim2pm-abstract">Abstract</a>
                <a class="badge grey waves-effect font-weight-light mr-1" href="/assets/pdf/COBRA_poster.pdf" target="_blank">Poster</a>
              </div>
      
              <div class="col mt-2 p-0">
                <div id="cobra_poster" class="collapse">
                  <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
                    Systems biology methods often rely on correlations in gene expression profiles to infer co-expression networks, commonly used as input for gene regulatory network inference or to identify functional modules of co-expressed or co-regulated genes. While systematic biases, including batch effects, are known to induce spurious associations and confound differential gene expression analyses (DE), the impact of batch effects on gene co-expression has not been fully explored. Methods have been developed to adjust expression values, ensuring conditional independence of mean and variance from batch or other covariates for each gene. These adjustments have been shown to improve the fidelity of DE analysis. However, these methods do not address the potential for spurious differential co-expression (DC) between groups. Consequently, uncorrected, artifactual DC can skew the correlation structure, leading network inference methods that use gene co-expression to identify false, nonbiological associations, even when the input data is corrected using standard batch correction.
                    In this work, we demonstrate the persistence of confounders in covariance after standard batch correction using synthetic and real-world gene expression data examples. Subsequently, we introduce Co-expression Batch Reduction Adjustment (COBRA), a method for computing a batch-corrected gene co-expression matrix based on estimating a conditional covariance matrix. COBRA estimates a reduced set of parameters expressing the co-expression matrix as a function of the sample covariates, allowing control for continuous and categorical covariates. COBRA is computationally efficient, leveraging the inherently modular structure of genomic data to estimate accurate gene regulatory associations and facilitate functional analysis for high-dimensional genomic data.
                  </div>
                </div>
              </div>
              
            </div>
          </div>
        </div></li></ol>
  
        <!--JAHA-->
        <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
          <div class="col-sm-1 p-0 abbr">
            <a class="badge font-weight-bold danger-color-dark align-middle" style="width: 53px;" href="https://www.ahajournals.org/journal/jaha" target="_blank">
            JAHA
            </a>
          </div>
          
          <div class="col-sm-11 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">
            <div id="jaha" class="col p-0">
              <h5 class="title mb-0">Profiling Daily Life Performance Recovery in the Early Subacute Phase After Stroke Using a Graphical Modeling Approach
              </h5>
              <div class="author">
                <em> 
                    Janne M. Veerbeek;
                    Clemens Hutter;
                    <b>Soel Micheletti</b>;
                    Simone Riedi;
                    Enrico Bianchi;
                    Beatrice Ottiger;
                    Noortje Maaijwee;
                    Tim Vanbellingen;
                    Thomas Nyffeler
                </em>  
              </div>
  
              <div>
                <p class="periodical font-italic">
                Journal of the American Heart Association         
                </p>
              </div>
      
              <div class="col p-0">
                <a class="badge grey waves-effect font-weight-light mr-1" data-toggle="collapse" href="#jaha" role="button" aria-expanded="false" aria-controls="tec2023clim2pm-abstract">Abstract</a>
                <a class="badge grey waves-effect font-weight-light mr-1" href="https://www.ahajournals.org/doi/full/10.1161/JAHA.123.030472" target="_blank">Paper</a>
                <a class="badge grey waves-effect font-weight-light mr-1" href="https://github.com/rauwuckl/CorrelationGraph4StrokeRecovery" target="_blank">Code</a>
              </div>
      
              <div class="col mt-2 p-0">
                <div id="jaha" class="collapse">
                  <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
                    Laboratory-based assessments have shown that stroke recovery is heterogeneous between patients and af- fected domains such as motor and language function. However, laboratory-based assessments are not ecologically valid and do not necessarily reflect patientsâ€™ daily life performance. Therefore, we aimed to give an innovative view on stroke recovery by profiling daily life performance recovery across domains in patients with early subacute stroke and determine their inter- relatedness, taking stroke localization into account.
                    Daily life performance was observed at neurorehabilitation admission and weekly thereafter until dis- charge, using a scale containing 7 daily life domains. Graphical modeling was applied to investigate the conditional independ- ence between recovery of these domains depending on stroke localization. There were 592 patients analyzed. Four clusters of interrelated domains were identified within the first 6weeks poststroke. The first cluster included recovery in learning and applying knowledge, general tasks and demands, and domestic life. The second cluster comprised recovery in self-care and general tasks and demands. The third cluster included recovery in mobility and self-care; it incorporated interpersonal interac- tions and relationships in left supratentorial stroke, and learning and applying knowledge in right supratentorial stroke. The final cluster included only communication recovery.
                    Daily life recovery dynamics early poststroke show that although impairments in body functions are anatomically determined, their impact on performance is comparable. Second, some, but by no means all, domains show an interrelated recovery. Domains requiring cognitive abilities are especially interrelated and seem to be essential for concomitant recovery in mobility and domestic life.
                  </div>
                </div>
              </div>
              
            </div>
          </div>
        </div></li></ol>
  </div>
  <div class="content">
    <h1>Outreach</h1>
      <div class="row m-0 p-0">
        <div class="col-sm-11 p-0">
      
          <!--LAM-->
          <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
              
              <div class="col-sm-11 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">
                <div id="lam" class="col p-0">
                  <h5 class="title mb-0">Calcolo di superfici con il metodo Monte Carlo
                  </h5>
                  <div class="author">
                    <em> 
                      <b>Soel Micheletti</b>
                    </em>  
                  </div>
      
                  <div>
                    <p class="periodical font-italic">
                    During high school I had the opportunity of working on a stochastic simulation project. As a recognition to the results of the project (Odd Fellow Award at the National Competition by Schweizer Jugend Forscht and a First Award in the CS category at the Taiwan International Science Fair 2018), a summary of our work has been featured on a special edition of <a href="https://www.vsmp.ch/cmsi/Il_Volterriano.php">Il Volterriano</a>, the journal of the Swiss-Italian Mathematics Commission. 
                    </p>
                  </div>
          
                  <div class="col p-0">
                    <a class="badge grey waves-effect font-weight-light mr-1" href="/assets/pdf/LAM.pdf" target="_blank">Paper</a>
                  </div>
                  
                </div>
              </div>
          </li></ol></div>
        </div>
      </div>
  </div>

  <!-- Footer -->
  <footer>
    &copy; Copyright 2022 Soel Micheletti.
    
    
  </footer>

  <!-- Core JavaScript Files -->
  <script src="/assets/js/jquery.min.js" type="text/javascript"></script>
  <script src="/assets/js/popper.min.js" type="text/javascript"></script>
  <script src="/assets/js/bootstrap.min.js" type="text/javascript"></script>
  <script src="/assets/js/mdb.min.js" type="text/javascript"></script>
  <script async="" src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" integrity="sha384-GNFwBvfVxBkLMJpYMOABq3c+d3KnQxudP/mGPkzpZSTYykLBNsZEnG2D9G/X/+7D" crossorigin="anonymous"></script>
  <script src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML"></script>
  <script src="/assets/js/common.js"></script>


  <!--Accordion-->
  <script>
  var acc = document.getElementsByClassName("accordion");
  var i;
  
  for (i = 0; i < acc.length; i++) {
    acc[i].addEventListener("click", function() {
      this.classList.toggle("active");
      var panel = this.nextElementSibling;
      if (panel.style.display === "block") {
        panel.style.display = "none";
      } else {
        panel.style.display = "block";
      }
    });
  }
  </script>

  <!-- Scrolling Progress Bar -->
  <script type="text/javascript">
    $(document).ready(function() {
      var navbarHeight = $('#navbar').outerHeight(true);
      $('body').css({ 'padding-top': navbarHeight });
      $('progress-container').css({ 'padding-top': navbarHeight });
      var progressBar = $('#progress');
      progressBar.css({ 'top': navbarHeight });
      var getMax = function() { return $(document).height() - $(window).height(); }
      var getValue = function() { return $(window).scrollTop(); }   
      // Check if the browser supports the progress element.
      if ('max' in document.createElement('progress')) {
        // Set the 'max' attribute for the first time.
        progressBar.attr({ max: getMax() });
        progressBar.attr({ value: getValue() });
    
        $(document).on('scroll', function() {
          // On scroll only the 'value' attribute needs to be calculated.
          progressBar.attr({ value: getValue() });
        });

        $(window).resize(function() {
          var navbarHeight = $('#navbar').outerHeight(true);
          $('body').css({ 'padding-top': navbarHeight });
          $('progress-container').css({ 'padding-top': navbarHeight });
          progressBar.css({ 'top': navbarHeight });
          // On resize, both the 'max' and 'value' attributes need to be calculated.
          progressBar.attr({ max: getMax(), value: getValue() });
        });
      } else {
        var max = getMax(), value, width;
        var getWidth = function() {
          // Calculate the window width as a percentage.
          value = getValue();
          width = (value/max) * 100;
          width = width + '%';
          return width;
        }
        var setWidth = function() { progressBar.css({ width: getWidth() }); };
        setWidth();
        $(document).on('scroll', setWidth);
        $(window).on('resize', function() {
          // Need to reset the 'max' attribute.
          max = getMax();
          setWidth();
        });
      }
    });
  </script>

  <!-- Code Syntax Highlighting -->
  <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet">
  <script src="/assets/js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>

  <!-- Script Used for Randomizing the Projects Order -->
  <!-- <script type="text/javascript">
    $.fn.shuffleChildren = function() {
      $.each(this.get(), function(index, el) {
        var $el = $(el);
        var $find = $el.children();

        $find.sort(function() {
          return 0.5 - Math.random();
        });

        $el.empty();
        $find.appendTo($el);
      });
    };
    $("#projects").shuffleChildren();
  </script> -->

  <!-- Project Cards Layout -->
  <script type="text/javascript">
    var $grid = $('#projects');

    // $grid.masonry({ percentPosition: true });
    // $grid.masonry('layout');

    // Trigger after images load.
    $grid.imagesLoaded().progress(function() {
      $grid.masonry({ percentPosition: true });
      $grid.masonry('layout');
    });
  </script>

  <!-- Enable Tooltips -->
  <script type="text/javascript">
    $(function () {
      $('[data-toggle="tooltip"]').tooltip()
    })
  </script>
</body>
</html>
